pom.xml

```diff
@@ -0,0 +1,71 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>miniCalculator</groupId>
+	<artifactId>miniCalculator</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+
+
+	<dependencies>
```

Comment:
Remove all dependencies as they are unnecessary for this project.

Reply:
Done